### PR TITLE
Hypershift: rosa cli to create multi az hosted cluster

### PR DESCRIPTION
**Pull request by mukrishn**
### Description

- Multiple subnets
- Multi az
- User OIDC config
- Kube-burner ocp wrapper workload
- Updated metric collection on installation
- Included delete ELB on cleanup

### CODE REVIEW

File: dags/openshift_nightlies/manifest.yaml [hunk 1]
```diff
@@ -28,8 +28,8 @@ dagConfig:
     tag: 2.3.2
   dependencies:
     e2e_benchmarking:
-      repo: https://github.com/cloud-bulldozer/e2e-benchmarking.git
-      branch: master
+      repo: https://github.com/mukrishn/e2e-benchmarking.git
+      branch: ocp-wrapper
```
Comment (mukrishn) [inline dags/openshift_nightlies/manifest.yaml:32]: i will replace after testing and code-review

**Review by krishvoor — APPROVED** [review]
lgtm

File: dags/openshift_nightlies/config/benchmarks/hosted-control-plane-p75.json [hunk 2]
```diff
@@ -2,22 +2,15 @@
...
-                "LOG_LEVEL": "info",
-                "PLATFORM_ALERTS": "false",
-                "CLEANUP_WHEN_FINISH": "true",
-                "CLEANUP": "true"
+                "ITERATIONS": "75",
+                "LOG_LEVEL": "debug",
+                "EXTRA_FLAGS": "--churn-duration=1h --churn-percent=10 --churn-delay=30s"
             }
-        }
+        }
```
Comment (krishvoor) [inline dags/openshift_nightlies/config/benchmarks/hosted-control-plane-p75.json:14]: Minor nits: Extra spaces

File: dags/openshift_nightlies/scripts/run_benchmark.sh [hunk 3]
```diff
@@ -27,14 +27,14 @@ setup(){
     curl -sS https://mirror.openshift.com/pub/openshift-v4/clients/ocp/latest/openshift-client-linux.tar.gz | tar xz oc
 
     export PATH=$PATH:$(pwd)
-
+
```
Comment (krishvoor) [inline dags/openshift_nightlies/scripts/run_benchmark.sh:30]: Minor nits: Extra spaces

**Review by krishvoor — APPROVED** [review]
lgtm

**Comment by mukrishn** [comment]
Re-login issue is handled, it fails for 1 or 2 re-attempts and works later on. http://pastebin.test.redhat.com/1103837

**Review by dry923 — APPROVED** [review]
LGTM but fix the repo so we can merge :-)

**Pull request merged**